library/ZendService/Twitter/Twitter.php

```diff
@@ -318,7 +336,8 @@ public function accountVerifyCredentials()
     {
         $this->init();
         $response = $this->get('account/verify_credentials');
-        return new Response($response);
+        $this->twitterResponse->populate($response);
```

Comment:
Hi! :)

This change was put into place - in all the place - so that it could be unit tested. Now instead of creating a Response object, we are injecting one and then populating it. This makes it possible to test.

This applied to all the ones below too. 

Cheers!
=C=